views/map_animator.py

```diff
@@ -209,9 +209,9 @@ def update_results(self):
 
             provider = layer.dataProvider()
 
-            values = ds.get_values_timestamp(parameter[0], timestamp_nr)
+            values = ds.get_values_by_timestamp(parameter[0], timestamp_nr)
```

Comment:
This part is not working (existing bug), should be parameter instead of parameter[0]